coderedcms/models/wagtailsettings_models.py

```diff
@@ -118,6 +120,15 @@ class Meta:
         related_name='favicon',
         verbose_name=_('Favicon'),
     )
+    navbar_chooser = models.ForeignKey(
```

Comment:
I think for this to be more useful it should be a ManyToMany field (allow them to select multiple navbars and footers). An it would then render them in order. This is similar to the current functionality where it renders all of the navbars.

To accomplish this you might need to use modelcluster/orderable, which lets you select multiple and rearrange the order of them. This is similar to how we have multiple carousel slides on a carousel snippet.